src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,194 @@
+                            System.out.println(todo.get(todo_index).toString());
+                            save += todo.get(todo_index).toString()+"\n";
+                            writeToFile("data/duke.txt",save);
+                            todo_index += 1;
+                        } catch (DukeException e) {
+                            System.out.println("____________________________________________________________");
+                            System.out.println("OOPS!!! The description of a deadline must have /by.");
+                            System.out.println("____________________________________________________________");
+                        } catch (StringIndexOutOfBoundsException e) {
+                            System.out.println("____________________________________________________________");
+                            System.out.println("OOPS!!! The description of a deadline cannot be empty.");
+                            System.out.println("____________________________________________________________");
+                        }
+
+                    } else if (input.startsWith("event")) {
+                        int descriptionIndex = 6;
+                        String task = "event";
+                        try {
+                            ExceptionTry(task, input);
+                            String description = input.substring(input.indexOf("event") + descriptionIndex, input.indexOf("/at") - 1);
+                            String at = input.substring(input.indexOf("/at") + 4);
+                            todo.add(new Event(description, at));
+                            System.out.println("____________________________________________________________");
+                            System.out.println("Got it. I've added this task: ");
+                            System.out.println(todo.get(todo_index).toString());
+                            save += todo.get(todo_index).toString()+"\n";
+                            writeToFile("data/duke.txt",save);
+                            todo_index += 1;
+                        } catch (DukeException e) {
+                            System.out.println("____________________________________________________________");
+                            System.out.println("OOPS!!! The description of a event must have /at.");
+                            System.out.println("____________________________________________________________");
+                        } catch (StringIndexOutOfBoundsException e) {
+                            System.out.println("____________________________________________________________");
+                            System.out.println("OOPS!!! The description of a event cannot be empty.");
+                            System.out.println("____________________________________________________________");
+                        }
+
+                    }else if(input.startsWith("delete")){
```

Comment:
Please maintain spacing between the parenthesis and the conditional statements like: `} else if (...`